beetmoverscript/data/maven_beetmover_task_schema.json

```diff
@@ -0,0 +1,97 @@
+{
+    "title": "Taskcluster beetmover task maven schema",
+    "type": "object",
+    "properties": {
+        "dependencies": {
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "type": "string"
+            }
+        },
+        "payload": {
+            "type": "object",
+            "properties": {
+                "upload_date": {
+                    "type": "number"
+                },
+                "build_number": {
+                    "type": "number"
+                },
+                "locale" : {
+                    "type" : "string"
+                },
+                "maxRunTime" : {
+                    "type" : "number"
+                },
+                "version": {
```

Comment:
Nit: I see most of the properties defined at this level isn't used in https://tools.taskcluster.net/groups/cu2TBeH2RWmzmaI49iX5XQ/tasks/BlnsxnNBSjW-CWUExDneOQ/details. I think we should clean this payload schema a little bit.

Reply:
I would but they are still used by geckoview.

Reply:
Ah my bad, I thought it was just used by android-components. In this case, could you put 
* https://github.com/mozilla-releng/beetmoverscript/pull/163/files#diff-14174189739af807ebb8ef15d2bf2f64L59
* and https://github.com/mozilla-releng/beetmoverscript/pull/163/files#diff-14174189739af807ebb8ef15d2bf2f64L84

back in ` beetmoverscript/data/beetmover_task_schema.json`?

Reply:
Ah, makes total sense. More security in checking the schema! Those are now reverted, good catch!